collection-scripts/gather_ceph_resources

```diff
@@ -384,6 +383,28 @@ for ns in $namespaces; do
                     } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${image}".part &
                     pids_rbd+=($!)
                 done
+                # Collecting mirror image status for provider mode with rados namespace
+                rados_namespaces=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd -p $bp namespace ls") 
+                for rns in $rados_namespaces; do
+                    {
+                        printf "Collecting mirror pool status for: %s\n" "${bp}"
+                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool status --verbose $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-status-"${bp}"-debug.log
+                    } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_status &
+                    pids_rbd+=($!)
+                    {
+                        printf "Collecting mirror pool info for: %s\n" "${bp}"
+                        timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log
+                    } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info &
+                    pids_rbd+=($!)
+                    images=$(timeout 60 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp --namespace ${rns}")
+                    for image in $images; do
+                        dbglogf "${CEPH_GATHER_DBGLOG}" "Printing information for image: ${image}"
+                        {
+                            timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror image status $bp/$image --namespace ${rns} --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-image-status-"${bp}"-"${image}"-debug.log
+                        } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status_"${image}".part &
```

Comment:
I'd expect the namespace to be included in the name of the "part" file, not just the image name -- so that these "part" files can be gathered into a single per-namespace result file later.